modules/server/main.tf

```diff
@@ -425,6 +432,32 @@ resource "google_storage_bucket" "cai_export" {
   depends_on = [null_resource.services-dependency]
 }
 
+resource "tls_private_key" "policy_library_sync_ssh" {
+  count       = "${var.policy_library_sync_enabled ? 1 : 0}"
+  algorithm   = "RSA"
+}
```

Comment:
Generate the SSH key so that Forseti can clone a private repo. The openssh public key is provided as output. I will update the docs to let the user know they will need to add this public key to their git account.